rasa/core/tracker_store.py

```diff
@@ -289,13 +289,19 @@ async def retrieve_full_tracker(
 
     async def stream_events(self, tracker: DialogueStateTracker) -> None:
         """Streams events to a message broker."""
+        offset = await self.number_of_existing_events(tracker.sender_id)
+        events = tracker.events
+        new_events = list(itertools.islice(events, offset, len(events)))
+
+        await self.stream_new_events(new_events, tracker.sender_id)
+
+    async def stream_new_events(self, new_events: List[Event], sender_id: Text) -> None:
```

Comment:
Should we mark this as an internal method by prefixing the method name with a `_`?